index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
          </div>
           <div class="support">
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-  don’t hestiate to contact me.</p>
+          don’t hestiate to contact me.</p>
           </div>
           <div class="social-media">
             <a href="#"><i class="fab fa-twitter social"></i></a>
             <a href="#"><i class="fab fa-linkedin social"></i></a>
             <a href="#"><i class="fab fa-medium-m social"></i></a>
-            <!-- <a href="#"><i class="fab fa-medium social"></i></a> -->
             <a href="#"><i class="fab fa-github social"></i></a>
             <a href="#"><i class="fab fa-angellist social"></i></a>
-
           </div>
-
           <div class="arrow">
             <button  class="arrow-button" ><i class="fas fa-chevron-down fa-2x"></i></button>
           </div>
         </section>
+   </div>
+   <div class="grid-container">
+       <h3>My Recent Works</h3>
+       <hr>
+
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
```

Comment:
- [ ] The button should and the button styling: Active, Hover, and Pressed. Apply it to other buttons on the page. [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)